app/questionnaire/path_finder.py

```diff
@@ -180,6 +180,10 @@ def get_blocks(self):
             if repeating_rule:
                 no_of_repeats = evaluate_repeat(repeating_rule, self.answer_store)
 
+            if no_of_repeats > 50:
+                logger.error('Excessive number of repeats found [%s] capping at 50', no_of_repeats)
+                no_of_repeats = 50
```

Comment:
Not sure this is what we want, if they have enter over 50, only repeat for 50. Ideally on a repeating feeding answer we should show an error if over 50 to make sure they just haven't made a mistake before this